### PR TITLE
Hotfix/fixing-nomenclatures

### PR DESCRIPTION
**Pull request by krishnaglodha**
Fixed nomenclatures of all functions as per review

### CODE REVIEW

File: src/geoserverx/_async/gsx.py [hunk 1]
```diff
@@ -132,33 +131,30 @@ async def get_all_workspaces(self) -> Union[WorkspacesModel, GSResponse]:
             results = self.response_recognise(responses.status_code)
```
Comment (ricardogsilva) [inline src/geoserverx/_async/gsx.py:131]: Missed this one `responses`

File: src/geoserverx/_async/gsx.py [hunk 2]
```diff
@@ -132,33 +131,30 @@ async def get_all_workspaces(self) -> Union[WorkspacesModel, GSResponse]:
...
+            headers=self.headers,
         )
-        results = self.response_recognise(responses.status_code)
+        results = self.response_recognise(response.status_code)
         return results
 
-    # Get vector stores in specific workspaces
     async def get_vector_stores_in_workspaces(self, workspace: str) -> DataStoresModel:
         Client = self.http_client
         responses = await Client.get(f"workspaces/{workspace}/datastores")
```
Comment (ricardogsilva) [inline src/geoserverx/_async/gsx.py:160]: And this one `responses` - please check whether there are more, better to change them all at once if we can.

File: src/geoserverx/_async/gsx.py [hunk 3]
```diff
@@ -132,33 +131,30 @@ async def get_all_workspaces(self) -> Union[WorkspacesModel, GSResponse]:
...
         Client = self.http_client
-        responses = await Client.get(f"workspaces/{workspace}")
-        if responses.status_code == 200:
-            return WorkspaceModel.model_validate(responses.json())
+        response = await Client.get(f"workspaces/{workspace}")
+        if response.status_code == 200:
+            return WorkspaceModel.model_validate(response.json())
         else:
-            results = self.response_recognise(responses.status_code)
+            results = self.response_recognise(response.status_code)
```
Comment (ricardogsilva) [inline src/geoserverx/_async/gsx.py:140]: Since we are at it, let's change `results` to `result` too

**Review by ricardogsilva — COMMENTED** [review]
Some changes still left to do, but almost done

File: src/geoserverx/_async/gsx.py (hunk 2 above)
Comment (krishnaglodha) [reply src/geoserverx/_async/gsx.py:160]: these will be multiple, since it will return multiple vector stores

**Pull request merged**